common/production_methods/07_government.txt

```diff
@@ -523,6 +523,59 @@ pm_urban_academia = {
 			country_prestige_add = 1
 		}
 	}
+	
+}
+pm_placeholder_1 = {
+	texture = "gfx/interface/icons/production_method_icons/government_run.dds"
+	building_modifiers = {
+		
+		level_scaled = {
+			building_employment_academics_add = 25
+            building_employment_clerks_add = 50
+            building_employment_laborers_add = 75
+		}
+		workforce_scaled = {
+			goods_input_paper_add = 0.20
+			goods_input_services_add = 0.20
+		}	
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_education_access_add = 0.001
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_prestige_add = 1
+		}
+	}
+	
+}
+pm_placeholder_2 = {
+	texture = "gfx/interface/icons/production_method_icons/government_run.dds"
+	building_modifiers = {
+		
+		level_scaled = {
+			building_employment_academics_add = 25
+            building_employment_clerks_add = 50
+            building_employment_laborers_add = 75
+		}
+		workforce_scaled = {
+			goods_input_paper_add = 0.20
+			goods_input_services_add = 0.20
+		}	
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_education_access_add = 0.001
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_prestige_add = 1
+		}
+	}
+	
```

Comment:
Formatting, basically a bunch of things

Reply:
Also why is this in a vanilla file like that? should be in babylon government.

Reply:
All our new university Pms were in there. Should we move all of them then?

Reply:
Yes, I think it was overlooked the previous time when I reviewed it. In order to keep it as compatible as possible with vanilla patches, and for our own sanity, it's best to put them in a separate file.